client/profile-wizard/steps/theme/index.js

```diff
@@ -229,6 +230,7 @@ class Theme extends Component {
 								isPrimary
 								onClick={ () => this.onChoose( theme, 'card' ) }
 								isBusy={ chosen === slug }
+								disabled={ isUpdatingProfileItems }
```

Comment:
Maybe we should disable all buttons when `chosen` is not `null` as well.

Reply:
@joshuatf Thank you for the catch!  Updated and test. It worked as well. Changed in https://github.com/woocommerce/woocommerce-admin/pull/5704/commits/e99d88f131cfcd8715de3e87936731877ee8940a